app/models/CSVModels.js

```diff
@@ -0,0 +1,26 @@
+import csv from 'csvtojson';
+import mongoose from '../services/mongoose.js';
+
+const name = 'inspiration'
+
+// using the one csv in the repo for now.
+// later, let this be variable.
+const file_path = `./assets/decks/${name}/${name}.csv`
+
+export const rows = await csv().fromFile(file_path);
+
+const schema_data = {}
+
+for (var header in rows[0]) {
+    schema_data[header] = String;
+}
+
+// TODO: What is this for?
+const ObjectId = mongoose.Schema.Types.ObjectId;
```

Comment:
This is a Mongoose schema "type". You don't need it unless you're storing a reference to another collection, as in:

```
const bookSchema = new Schema({
  title: 'String',
  author: { type: 'ObjectId', ref: 'Person' }
});
const Book = mongoose.model('Book', bookSchema);
```